templates/isv.html

```diff
@@ -0,0 +1,118 @@
+{% extends "base_layout.html" %}
+
+{% block title %}MicroK8s for ISV Campaign{% endblock %}
+
+{% block meta_description %}MicroK8s is the easiest way to leverage the benefits of a modern, Kubernetes-based architecture with simple customer adoption for cloud native software development companies.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1pcUx3oJ4ZcgRG6nGv6kzHSf9tFZZnaTK1jzecKFZxGo/edit#{% endblock %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1>MicroK8s for software developers</h1>
+      <p class="p-headind--2">We deliver containers that you develop</p>
+      <a class="p-button--positive" href="/contact-us">Get in touch</a>
+    </div>
+    <div class="col-4">
+      <img src="https://assets.ubuntu.com/v1/f145e15a-K8s_Multi-cloud_solid.svg" alt="" width="300" height="178">
```

Comment:
I can't see the image here on the copy doc.

Reply:
I've just added it in, as it was only mentioned a conversation